README.md

```diff
@@ -60,6 +60,110 @@ which makes it easy for development teams to get started.
   libraryDependencies += "com.google.cloud.bigtable" % "bigtable-hbase-1.x" % "2.0.0-alpha2"
   ```
 [//]: # ({x-version-update-end})
+
+* Connecting to Cloud Bigtable
```

Comment:
We have examples here: https://cloud.google.com/bigtable/docs/hbase-connecting. I think that is the more appropriate place to add more?

Reply:
(and instead we can link to the docs via the README)

Reply:
Sounds good!